### ISSUE TITLE
Ignore destructive changes for deltas in Workflows

### ISSUE THREAD

**Issue by scolladon** [scolladon/sfdx-git-delta#369]
### Discussed in https://github.com/scolladon/sfdx-git-delta/discussions/367

<div type='discussions-op-text'>

<sup>Originally posted by **jdgirald** October 11, 2022</sup>

When getting destructive changes for Workflows, the subtypes are extracted and added to the destructiveChanges.xml file. For example
```
<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>Opportunity.qualtrics__Qualtrics</members>
        <name>WorkflowOutboundMessage</name>
    </types>
    <types>
        <members>Opportunity.Update account</members>
        <name>WorkflowRule</name>
    </types>
    <types>
        <members>Opportunity.qualtrics__Qualtrics Example</members>
        <name>WorkflowRule</name>
    </types>
    <version>54.0</version>
</Package>
```

Is it possible to ignore subtypes? For example, I want to ignore destructive changes for the `Opportunity.Update account` workflow rule but leave the other components.

I have tried adding different values for the `ignore ` and ` ignore-destructive` options but it seems to only work with file names, which in this case is going to ignore all the individual components.

Thanks
</div>

**Comment by scolladon** [comment]
TL;DR
We are not going to deal with this issue as Workflow are going to be deprecated soon
